cmd/pool/order.go

```diff
@@ -210,6 +210,42 @@ func promptForConfirmation(msg string) bool {
 	}
 }
 
+// parseMinChanAmount parses and validates the value for the `min_chan_amt`
+// parameter.
+func parseMinChanAmount(ctx *cli.Context, amt uint64) (btcutil.Amount, error) {
+	minChanAmt := btcutil.Amount(ctx.Uint64("min_chan_amt"))
+
+	auctionType := ctx.String("auction_type")
+	if auctionType == auctionTypeInboundLiquidity {
+		if minChanAmt == 0 {
```

Comment:
Can we preserve the comment about the default of `10%` here please? Just to give this some context.